t/t7502-commit-porcelain.sh

```diff
@@ -262,13 +262,57 @@ test_expect_success 'cleanup commit messages (scissors option,-F,-e, scissors on
 
 	echo >>negative &&
 	cat >text <<-\EOF &&
+
 	# ------------------------ >8 ------------------------
 	to be removed
 	EOF
 	git commit --cleanup=scissors -e -F text -a --allow-empty-message &&
 	git cat-file -p HEAD >raw &&
 	sed -e "1,/^\$/d" raw >actual &&
-	test_must_be_empty actual
+	test_must_be_empty actual 
+'
+
+test_expect_success 'cleanup commit messages (scissors option,-F,-e, CR/LF line endings)' '
+
+	echo >>negative &&
+	test_config core.editor unix2dos &&
+	cat >text <<-\EOF &&
+
+	# to be kept
+
+	   # ------------------------ >8 ------------------------
+	# to be kept, too
+	# ------------------------ >8 ------------------------
+	to be removed
+	# ------------------------ >8 ------------------------
+	to be removed, too
+	EOF
+
+	cat >expect <<-\EOF &&
+	# to be kept
+
+	   # ------------------------ >8 ------------------------
+	# to be kept, too
```

Comment:
Oh, so I was fooled by the comment. That line was not even meant to be kept, yet the comment above it said "to be kept, too"? Why confuse the reader? Please don't.

Reply:
Ah, those comments were not meant to refer to the following lines? Now I understand what you were thinking. And a comment like "# keep this comment", "# keep this comment, too" would have helped me understand your intention a lot better.

Reply:
As above, would you like me to change the language in the [existing test](https://github.com/git-for-windows/git/blob/a98b073ae2b993892641bb1eb6ce707cc673d9ad/t/t7502-commit-porcelain.sh#L239-L246)? I *really* don't mean to bicker, cloning an existing test seemed like the safest thing to do.